api/src/main/resources/workbench.yaml

```diff
@@ -66,6 +66,25 @@ paths:
           schema:
             $ref: "#/definitions/BugReport"
 
+  # TODO(dmohs): If the username is not present, this response with 500 Server Error. It should
```

Comment:
I think instead this should just return a 200 with "yes" or "no". It's not really an error for the user to type in a name that's already taken, or to type in a name that isn't taken... returning a 400 is going to be confusing from a monitoring standpoint.

Reply:
My comment here was very unclear. Make sense now?

Reply:
Ah, yes, thanks!